src/cli/task.rs

```diff
@@ -182,6 +193,42 @@ pub struct Args {
     pub manifest_path: Option<PathBuf>,
 }
 
+fn print_heading(value: &str) -> io::Result<()> {
+    let bold = console::Style::new().bold();
+    let mut writer = tabwriter::TabWriter::new(stdout());
+    writeln!(
+        writer,
+        "{}\n{:-<2$}",
+        bold.apply_to(value),
+        "",
+        value.len() + 4
+    )?;
+    writer.flush()?;
+    Ok(())
+}
```

Comment:
The tabwriter is not required here. 

Its also better to write "user" focused info to `eprintln` e.g.:
```rust
fn print_heading(value: &str){
    let bold = console::Style::new().bold();
    eprintln!(
        "{}\n{:-<2$}",
        bold.apply_to(value),
        "",
        value.len(),
    );
}
```

Reply:
thanks! was wondering the same since your previous comment was on this block so i changed both.
 
should the other `println` that i have added also be changed to `eprintln` ? e.g `println!("{}", formatted);`